Fabric.Authorization.Persistence.SqlServer/Services/InMemorySecurityContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Fabric.Authorization.Persistence.SqlServer.Stores.EDW;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fabric.Authorization.Persistence.SqlServer.Services
+{
+    public class InMemorySecurityContext : SecurityContext
+    {
+        public InMemorySecurityContext()
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            //optionsBuilder.UseInMemoryDatabase(databaseName: ConnectionStrings.AuthorizationDatabase);
```

Comment:
Can we make a valid in memory context here so we can test this appropriately in our integration tests?